packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -241,30 +221,29 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
      */
     constructor(
         uint256 _emissionRate,
-        uint256 _spamDeletionProposalBond,
         uint64 _unstakeCoolDown,
         uint64 _phaseLength,
-        uint64 _minRollToNextRoundLength,
+        uint32 _deleteAfterRollCount,
         uint256 _gat,
         uint64 _startingRequestIndex,
         address _votingToken,
         address _finder,
         address _slashingLibrary,
         address _previousVotingContract
     ) Staker(_emissionRate, _unstakeCoolDown, _votingToken) {
-        voteTiming.init(_phaseLength, _minRollToNextRoundLength);
+        voteTiming.init(_phaseLength);
         require(_gat < IERC20(_votingToken).totalSupply() && _gat > 0);
         gat = _gat;
         finder = FinderInterface(_finder);
         slashingLibrary = SlashingLibraryInterface(_slashingLibrary);
         previousVotingContract = OracleAncillaryInterface(_previousVotingContract);
-        setSpamDeletionProposalBond(_spamDeletionProposalBond);
+        setDeleteAfterRollCount(_deleteAfterRollCount);
 
         // We assume indices never get above 2^64. So we should never start with an index above half that range.
         require(_startingRequestIndex < type(uint64).max / 2);
 
         assembly {
-            sstore(priceRequestIds.slot, _startingRequestIndex)
+            sstore(resolvedPriceRequestIds.slot, _startingRequestIndex)
```

Comment:
Do we need to set this at all anymore? Are these referenced by their index? Would the user/system care about their index/uniqueness?

Reply:
setting this as we did before primarily gives us the ability to offset the request "number" to shift a new deployment to _start_ after a previous deployment. for example we could set the new dvm to start after the previous DVM.

we could get rid of this but it seems pretty nice from a usability and continuity perspective.

Reply:
that's true and is a good point, especially seeing that the ordering of resolution is not guaranteed to be the same as the order in which it is enqueued, like it is within the previous voting system. it might make sense in that case to totally remove this feature and have no offsets.